### ISSUE TITLE
Question about the evidential loss term in Eq.(1)

### ISSUE THREAD

**Issue by HeimingX** [Cogito2012/DEAR#1]
Dear authors,

Thanks a lot for the interesting paper and open-resourced repo.

It is awesome to incorporate the evidential deep learning (EDL) trick into action recognition task. But I have a question about the EDL loss term, i.e., Eq. (1) in DEAR paper. In this repo, the EDL loss with log function is set to be the default choice for running DEAR algorithm. I wonder if the digamma function can work properly? For me, I have conducted some experiments about training a EDL loss with digamma function on CIFAR100 dataset (based on the implementation in this repo) and I just found that the EDL loss term decreases quite hard and slow and the model does not get well optimized. I am curious if it happens to you?

Look forward to you response.

Best,
Haiming

**Comment by Cogito2012** [comment]
@HeimingX Thanks for your interest in this work!
I encountered the same problem as yours. In my early trial, the digamma form was set as the default, but the model can hardly be trained no matter how I change the evidence function or tune training hyperparameters. I guess it is more difficult in optimization with the digamma function than the logarithm.

For the image dataset, I guess you could try the original implementation of EDL in [https://github.com/dougbrion/pytorch-classification-uncertainty](https://github.com/dougbrion/pytorch-classification-uncertainty).

**Comment by HeimingX** [comment]
Hi, thanks a lot for the prompt response and kind suggestions.
